docs/core/extensions/options.md

```diff
@@ -27,8 +27,8 @@ Create the following `TransientFaultHandlingOptions` class:
 
 An options class:
 
-- It allows for either parameterless constructors, or for constructors where all parameters have default values
-- Properties can be read-write, or read-init only
+- Must be a non-abstract class with either a public parameterless constructor, or a public constructor where all parameters have default values.
+- All bindable properties must be read-write, or init only. Fields are **not** bound.
```

Comment:
You changed "public" to "bindable". Was this intentional? Are any other visibilities except "public" supported here?

Reply:
My take: I think the most precise term would be "accessible". In practice, I think that means public properties. I suppose an option type could have additional properties that are protected or internal, but aren't bound. That's true, but not very helpful.

Reply:
@BillWagner So would you prefer "accessible" or "public"? I personally would tend to "public" because that's what developers will write (and "accessible" is a little muddy in my opinion because it depends on the "point-of-view"; e.g. private members are accessible from within a class but they're not what this paragraph is getting at).

Reply:
There actually seems to be an option to bind non public too, https://github.com/dotnet/runtime/blob/master/src/libraries/Microsoft.Extensions.Configuration.Binder/src/BinderOptions.cs.